src/docs/truffle/reference/configuration.md

```diff
@@ -51,6 +51,135 @@ Any of the following solutions will remedy this issue:
 
 ## General options
 
+### SolidityLog
+
+Like javascript's console.log, you can utilize the same concept to print state
+of your contract during execution. This log will appear when you use `truffle
+test` or `truffle develop`. To use it you would import `truffle/console.sol`
+into your solidity contract and use it like you would in JavaScript. See the 
+[Ganache documentation](https://github.com/trufflesuite/ganache/tree/develop/src/chains/ethereum/console.log#ganacheconsolelog) for details.
```

Comment:
I don't think we should link here since the examples here are more thorough than the examples in `@ganache/console.log`, and the ganache docs have a lot of irrelevant (to Truffle users) additional information and documentation.